### ISSUE TITLE
Applicant - Relative modals on job application

### ISSUE THREAD

**Issue by gobyrne** [GCTC-NTGC/TalentCloud#372]
# Description
Tie in the modals that Josh made for the Job application so that applicants can add references and work samples to their skills directly on the job application.

**Comment by gobyrne** [comment]
I think it's a duplicate of #463

**Comment by tristan-orourke** [comment]
Actually, this isn't complete, and its also not a duplicate.

**Comment by gobyrne** [comment]
Can you please elaborate here on in the description?

**Comment by gobyrne** [comment]
Waiting on front-end framework (#701)

**Comment by Khristophor** [comment]
Created the Modal in React as a component, but in order to properly implement it I think we should do some heavy refactoring of the relevant applicant sections (Application Skills and Applicant Profile). This will involve adding API routes, creating the required React components for the containers, creating a Skill component and retrieving the data.

**Comment by yonikid15** [comment]
My bad. Accidentally closed the issue.
